rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -40,9 +43,15 @@ Create categories to group processes. Assign a process to a category during eval
 . Enter a name in the *Create a category* window.
 . Click *OK*.
 
-=== List Applications
+=== Manage Applications
+
+In the view *Process Applications*, list all applications required for the execution of processes. In the view *Process Management*, assign required applications to processes at process creation or phase transition. In the module *Bot Management*, assign applications installed on a computer a bot is installed on to that bot.
```

Comment:
I had to reread the last sentence a couple of times to understand it. Could it be simplified to just: 

In the module *Bot Management*, assign applications to a bot installed on a computer.

Reply:
Yes, it should be simplified, but no, that is too litlle information:)